lib/Service/PageService.php

```diff
@@ -763,8 +769,9 @@ public function copy(int $collectiveId, int $id, ?int $parentId, ?string $title,
 		if (null !== $newFile = $this->moveOrCopyPage($collectiveFolder, $file, $parentId, $title, true)) {
 			$file = $newFile;
 		}
+		$slug = $this->generateSlugForPage($title ?: $pageInfo->getTitle(), $file);
 		try {
-			$this->updatePage($collectiveId, $file->getId(), $userId, $pageInfo->getEmoji(), $pageInfo->isFullWidth());
+			$this->updatePage($collectiveId, $file->getId(), $userId, $pageInfo->getEmoji(), null, $slug);
```

Comment:
```suggestion
			$this->updatePage($collectiveId, $file->getId(), $userId, $pageInfo->getEmoji(), $pageInfo->isFullWidth(), $slug);
```

I guess you removed `$pageInfo->isFullWidth()` by accident.